python/test/context/test_context.py

```diff
@@ -237,6 +241,27 @@ def test_rho_to_eps():
     print(dp_sum.release())
 
 
+def test_approx_to_approx_zCDP():
+    context = dp.Context.compositor(
+        data=[1, 2, 3],
+        privacy_unit=dp.unit_of(contributions=1),
+        privacy_loss=dp.loss_of(epsilon=1.0, delta=1e-6),
+        split_evenly_over=1,
+    )
+
+    azcdp_measure = dp.approximate(dp.zero_concentrated_divergence())
+    context_azcdp = context.query().compositor(3, output_measure=azcdp_measure, alpha=0.3).release()
+
+    dom, met = context_azcdp.accountant.input_space
+    context_azcdp(dp.m.make_user_measurement(
+        dom, met,
+        azcdp_measure,
+        lambda x: x,
+        # remaining rho, and catastrophic delta
+        lambda _: (.006, 1e-6 * .3 / 3)
+    ))
```

Comment:
Make some kind of assertion? Maybe just about the shape of the response, if the actual values are indeterminate.